.gitignore

```diff
@@ -7,4 +7,4 @@ terraform.tfstate.backup
 auto_backend.tf
 .env/
 __pycache__/
-
+environ.sh
```

Comment:
Good idea, it's better than copy and paste the environment variables on the terminal